### ISSUE TITLE
Compatibility with NumPy 1.20

### ISSUE THREAD

**Issue by amotl** [PyTables/PyTables#875]
Dear @avalentino and @eumiro,

this is just a little heads up coming from [1]. NumPy 1.20 was released yesterday and apparently, PyTables will need some adjustments.
```python
File "/home/runner/work/PyTables/PyTables/tables/atom.py", line 339, in tables.atom.Atom.from_sctype
Failed example:
    Atom.from_sctype('Float64')
Exception raised:
    Traceback (most recent call last):
      File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/doctest.py", line 1336, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest tables.atom.Atom.from_sctype[3]>", line 1, in <module>
        Atom.from_sctype('Float64')
      File "/home/runner/work/PyTables/PyTables/tables/atom.py", line 346, in from_sctype
        raise ValueError(f"unknown NumPy scalar type: {sctype!r}")
    ValueError: unknown NumPy scalar type: 'Float64'
```

With kind regards,
Andreas.

[1] https://github.com/daq-tools/PyTables/runs/1800879091?check_suite_focus=true#step:11:3260

**Comment by eumiro** [comment]
Thank you, @amotl. This is something @keszybz fixed in several cases in #862, but at that time the `numpy` imports were quite heterogeneous. In between I consolidated these imports in #864, but of course, this does not use `np.`, so it escaped our attention.

Would you like to create a PR with all fixes?

**Comment by amotl** [comment]
Hi Miroslav,

thanks for your quick response. I am currently still busy on trying to push #872 forward and would appreciate if you or @keszybz can follow up on this.

With kind regards,
Andreas.

**Comment by eumiro** [comment]
No problem, I'll have a look at it. There are problems in #874 anyway, so it is a good moment to fix it and to add doctests to GHA CI.

**Comment by amotl** [comment]
This issue will probably be fixed through #866 and #876, right?

**Comment by avalentino** [comment]
This should be fixed by #866 and #862. Issue #876 should only fix future deprecations.
Closing.
@amotl please re-open if you still have problems with this.

**Comment by ArchangeGabriel** [comment]
To be precise, this is fixed in https://github.com/PyTables/PyTables/commit/93a3272b8fe754095637628b4d312400e24ae654 (in case anyone has to backport it like me).
